template/main.go.ejs

```diff
@@ -4,12 +4,12 @@ package main
 
 <% schema.exports.forEach(ex => { %>
 <% if (hasComment(ex)) -%>	
-// <%= formatCommentBlock(ex.description, "// ") %>
+// <%- formatCommentBlock(ex.description, "// ") %>
```

Comment:
these don't html encode the comments. we may want to consider always using `%-` everytwhere actually since we're not using html. I can change this when i get back to refactor.

Reply:
TIL - thank you!